packages/sdk/src/hooks/useEnvWeb3.ts

```diff
@@ -22,9 +22,12 @@ export const defaultRPC = {
     [SupportedChainId.FUSE]: 'https://rpc.fuse.io'
 }
 
-export const getRpc = (chainId: number):string => {
+export const getRpc = (chainId: number, network?: string):string => {
   const rpcs = localStorage.getItem('GD_RPCS')
   if (!rpcs) return 'https://rpc.fuse.io'
+  if (network === ('staging' || 'fuse')){
+    return 'https://rpc.fuse.io'
+  }
```

Comment:
1. this should not be required
2. you cant write `=== ('staging' || 'fuse')` it wont do what you expect. it will always check vs 'staging'

Reply:
resolved now